README.md

```diff
@@ -42,8 +42,8 @@ pipx run --spec phylum phylum-init <options>
 pipx run --spec phylum phylum-ci <options>
 ```
 
-These installation methods require Python 3.7+ to run. For a self contained environment, consider using the Docker
-image as described below.
+These installation methods require Python 3.7+ to run. The application requires a POSIX compliant shell in which to run.
```

Comment:
> The application requires a POSIX compliant shell in which to run.

Where does that requirement come from? Are we using `shell=True` somewhere?

Reply:
`shell=True` is not used anywhere in the code base currently...and will be enforced going forward when #14 is completed, by using tools such as `bandit` which flags such use as a HIGH severity finding.

The requirement comes from a warning about using `shlex.quote` [from the Python docs](https://docs.python.org/3/library/shlex.html#shlex-quote-warning)
  > Warning The shlex module is only designed for Unix shells.
The [quote()](https://docs.python.org/3/library/shlex.html#shlex.quote) function is not guaranteed to be correct on non-POSIX compliant shells or shells from other operating systems such as Windows. Executing commands quoted by this module on such shells can open up the possibility of a command injection vulnerability. Consider using functions that pass command arguments with lists such as [subprocess.run()](https://docs.python.org/3/library/subprocess.html#subprocess.run) with `shell=False`.

Reply:
That warning would apply if we were passing the string to a shell... But we are just passing it into `shlex.split()`, which is POSIX compliant.

Our code should still work fine on Windows.

Reply:
`shlex` is gone and so is this restriction.